src/modules/Routes/Categories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <button type="button">Check status</button>
+    </div>
```

Comment:
✅ Should display only a button with text "Check status" 👍🏻

Reply:
Hi, this is a button